src/libs/ReportUtils.js

```diff
@@ -766,6 +766,16 @@ function getReport(reportID) {
     return lodashGet(allReports, `${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {}) || {};
 }
 
+/**
+ * Get the notification preference given a report
+ *
+ * @param {Object} report
+ * @returns {String}
+ */
+function getReportNotificationPreference(report) {
```

Comment:
Wouldn't it make more sense to only pass the reportID here given we have the allReports dict here?

Reply:
Yeah, I thought about this. However, decided to pass the `report` object, because most functions use `report` in **ReportUtils.js**. It is a kind of a consistency moment. But if we okay to call `getReport` inside `getReportNotificationPreference`, I would like to update it 🙂

Reply:
It looks like that - d3daf61. 
FYI: I reverted the commit until we decide the final solution.